fbchat/client.py

```diff
@@ -20,6 +20,8 @@ class Client(object):
     See https://fbchat.readthedocs.io for complete documentation of the API.
     """
 
+    ssl_verify = True
```

Comment:
I like the extra customization you sneaked in here ;), though I don't think we should say "debugging with Fiddler", just something like "debugging with a proxy"

Reply:
Done